collections/ansible_collections/cloudkit/templates/roles/ocp_4_17_small/meta/argument_specs.yaml

```diff
@@ -0,0 +1,34 @@
+argument_specs:
+  main:
+    options:
+      cluster_order:
+        type: dict
+        required: true
```

Comment:
do you think it's still worth to pass the `cluster_order`?

Reply:
I think templates should have access to the cluster order. We're using it right now e.g. to access `spec.nodeRequests`.